modules/sequenza/1.4/src/samtools.py

```diff
@@ -0,0 +1,112 @@
+import shlex
```

Comment:
Where is this script used? I don't think it is referenced in the Lofreq.smk or module config

Reply:
Oops. This shouldn't be here.

Reply:
I'll remove for final PR once I move everything to the new version directory